Makefile

```diff
@@ -244,7 +247,7 @@ set -e; \
 package=$(2)@$(3) ;\
 echo "Downloading $${package}" ;\
 rm -f $(1) || true ;\
-GOBIN=$(LOCALBIN) go install $${package} ;\
+GOBIN=$(LOCALBIN) go install -mod=readonly $${package} ;\
```

Comment:
Without `-mod=readonly`, go install fails in CI because of vendoring. This makes install ignore vendoring and install the dependency requested